Resources/Prototypes/Entities/Structures/Power/Generation/Singularity/collector.yml

```diff
@@ -38,6 +39,12 @@
         !type:CableDeviceNode
         nodeGroupID: HVPower
   - type: RadiationCollector
+    chargeModifier: 7500
+    radiationReactiveGases:
+      - reactantPrototype: Plasma
+        powerGenerationEfficiency: 1
+        reactantBreakdownRate: 0.0002
+        byproductPrototype: Tritium
```

Comment:
Doesn't tritium also need to be made a `RadiationReactiveGas` in order for output to level out as you suggested before?

Reply:
I originally intended tritium to also be useable, but it was brought up in the Discord discussion the possibility that you could just fill the tanks with cold tritium and call it a day, which goes against the entire point of this PR. I'd have to nerf tritium to the point of useless to discourage the behavior, so why both keeping it? All the numbers from the models in the PR description are from the pure usage of plasma

Reply:
I see, got it, thanks.